### DISCUSSION TITLE
[월드 1.1.5] 2D Sprite 패키지 지원하나요?

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#592] [5. Q&A]
지원하는 패키지 목록에는 2D Sprite는 없는데 이거 지원하나요?~
![image](https://user-images.githubusercontent.com/50188795/161964119-3609e73a-52e2-44c3-8e00-f3e940350057.png)

![image](https://user-images.githubusercontent.com/50188795/161964100-58b3e43b-240f-4b6f-b8b1-6794c0bede7d.png)

**Comment by Cha-SeongHun** [comment]
에디터상에서만 사용되는 스프라이트 에디터 패키지라 그런지 정상적으로 사용 가능했습니다
